inc/classes/subscriber/third-party/Hostings/class-litespeed-subscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace WP_Rocket\Subscriber\Third_Party\Hostings;
+
+use WP_Rocket\Logger\Logger;
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Subscriber for compatibility with Litespeed
+ *
+ * @since  3.4.1
+ * @author Soponar Cristina
+ */
+class Litespeed_Subscriber implements Subscriber_Interface {
+	/**
+	 * Subscribed events for Litespeed.
+	 *
+	 * @since  3.4.1
+	 * @author Soponar Cristina
+	 * @inheritDoc
+	 */
+	public static function get_subscribed_events() {
+		if ( ! isset( $_SERVER['HTTP_X_LSCACHE'] ) ) {
+			return [];
+		}
+
+		return [
+			'before_rocket_clean_domain' => 'rocket_litespeed_clean_domain',
+			'before_rocket_clean_file'   => [ 'rocket_litespeed_clean_file', 10, 1 ],
```

Comment:
`10` and `1` are the default values, so they can be omitted. The line can simply be written like this:
```php
'before_rocket_clean_file'   => 'rocket_litespeed_clean_file',
```
Anyway, it doesn't hurt to keep it like this :)

Reply:
Removed default values